Makefile

```diff
@@ -81,8 +82,11 @@ pcm-sensor-server.x: pcm-sensor-server.o $(COMMON_OBJS)
 libpcm.so: $(COMMON_OBJS) pcm-core.o
 	$(CXX) $(LDFLAGS) $(CXXFLAGS) -DPCM_SILENT -shared $^ $(LIB) -o $@
 
-c_example.x: c_example.o libpcm.so
-	$(CC) $^ -ldl -L./ -lpcm -Wl,-rpath,$(shell pwd) -o $@
+c_example.x: c_example.c libpcm.so
+	$(CC) -DPCM_DYNAMIC_LIB $< -ldl -Wl,-rpath,$(shell pwd) -o $@
+
+c_example_shlib.x: c_example.c libpcm.so
+	$(CC) $< -L./ -Wl,-rpath=$(shell pwd) -lpcm -o $@
```

Comment:
please use "," here instead of "=" to make it compile on MacOSX.

Background: https://stackoverflow.com/questions/26472590/ld-unknown-option-rpath-libs-linking-error-while-building-openframework/32400674

Reply:
Done.